App/src/views/NewRequests.js

```diff
@@ -0,0 +1,127 @@
+/** @jsx jsx */
+
+import React, { useState, useEffect } from "react";
+import { jsx } from "@emotion/core";
+import { Redirect } from "@reach/router";
+import Nabvar from "../components/Nabvar";
+import { requestsFetch, requestAdmin } from "../services/request";
+import RequestModal from "../components/RequestModal";
+import { useUser } from "../contexts/user";
+import Request from "../components/Request";
+import { users } from "../services/user";
+
+const Requests = () => {
+  const [requests, setRequests] = useState([]);
+  const [requestsAdmin, setRequestsAdmin] = useState([]);
+  const [frontdesks, setFrontdesks] = useState([]);
+
+  const [id, setId] = useState(0);
+  const user = useUser();
+
+  useEffect(() => {
+    async function fetchData() {
+      const response = await requestsFetch();
+      setRequests(response);
+    }
+    fetchData();
+  }, []);
+
+  useEffect(() => {
+    async function fetchData() {
+      const response = await requestAdmin();
+      setRequestsAdmin(response);
+    }
+    fetchData();
+  }, []);
+
+  useEffect(() => {
+    async function fetchData() {
+      const response = await users();
+      setFrontdesks(response);
+    }
+    fetchData();
+  }, []);
+
+  function findName(requester_id) {
+    return frontdesks.find(frontdesk => frontdesk.id === requester_id).name;
+  }
+
+  if (!user) return <Redirect to="login" noThrow />;
+  if (frontdesks.length === 0) return "Cargando...";
+  if (!requests.length === 0) return "Cargando...";
```

Comment:
Instead of hiding the whole UI when loading it is better to show the navbar and only hide the list of requests and show a `Loading...` message.

Reply:
I have a problem, I tried to do it, but it seems doesn't work 
Iḿ trying to apply the similitude of your code from new calendar  `          </Fragment>
    

        {workshiftList && (
          <Fragment>
            {forecast && (
              <tr>
                <td
                  colSpan={8}
                  css={{
                    backgroundColor: "#7B93DB",
                    color: "white",
                    textTransform: "uppercase",
                    padding: "5px 20px",
                    fontSize: 12
                  }}
                >
                  {Object.entries(workshiftList).length === 0
                    ? "Loading Shifts..."
                    : "Shifts"}
                </td>
              </tr>`
 What do you recommend me to do ?

Reply:
Thank you I already resolve it  ` return (
      <>
        <Nabvar />
        <h3
          css={{
            fontSize: "1em",
            color: "#35469C"
          }}
        >`